pytket/docs/index.rst

```diff
@@ -25,26 +26,30 @@ Windows. To install, run
     4. ``OPENBLAS="$(brew --prefix openblas)" pip install scipy``;
     5. ``pip install pytket``.
 
+If you have issues installing ``pytket`` please visit the `installation troubleshooting <https://cqcl.github.io/tket/pytket/api/install.html>`_ page.
+
 To use ``pytket``, you can simply import the appropriate modules into your python code or in an interactive Python notebook. We can build circuits directly using the ``pytket`` interface by creating a blank circuit and adding gates in the order we want to apply them.
 
 ::
 
     from pytket import Circuit
-    c = Circuit(2,2) # define a circuit with 2 qubits and 2 bits
-    c.H(0)           # add a Hadamard gate to qubit 0
-    c.Rz(0.25, 0)    # add an Rz gate of angle 0.25*pi to qubit 0
-    c.CX(1,0)        # add a CX gate with control qubit 1 and target qubit 0
-    c.measure_all()  # measure qubits 0 and 1, recording the results in bits 0 and 1
+
+    circ = Circuit(2,2) # define a circuit with 2 qubits and 2 bits
+    circ.H(0)           # add a Hadamard gate to qubit 0
+    circ.Rz(0.25, 0)    # add an Rz gate of angle 0.25*pi to qubit 0
+    circ.CX(1,0)        # add a CX gate with control qubit 1 and target qubit 0
+    circ.measure_all()  # measure qubits 0 and 1, recording the results in bits 0 and 1
 
 Some of the extension modules define :py:class:`Backend` s, allowing the circuits to be run on simulators or real quantum hardware. For example, ``pytket-qiskit`` grants access to the :py:class:`AerBackend` simulator which can sample from measurements.
 
 ::
 
     from pytket.extensions.qiskit import AerBackend
-    b = AerBackend()                            # connect to the backend
-    compiled = b.get_compiled_circuit(c)        # compile the circuit to satisfy the backend's requirements
-    handle = b.process_circuit(compiled, 100)   # submit the job to run the circuit 100 times
-    counts = b.get_result(handle).get_counts()  # retrieve and summarise the results
+
+    backend = AerBackend()                            # connect to the backend
+    compiled_circ = b.get_compiled_circuit(circ)      # compile the circuit to satisfy the backend's requirements
+    handle = backend.process_circuit(compiled_circ, 100)   # submit the job to run the circuit 100 times
```

Comment:
Please fix alignment.

Reply:
Thanks - forgot to fix this earlier. Its done now.